pages/Product.vue

```diff
@@ -148,16 +156,20 @@
           </div>
           <div class="col-xs-12 col-sm-5">
             <ul class="attributes p0 pt5 m0">
-              <product-attribute
+              <lazy-hydrate
                 :key="attr.attribute_code"
                 v-for="attr in getCustomAttributes"
-                :product="getCurrentProduct"
-                :attribute="attr"
-                empty-placeholder="N/A"
-              />
+                when-idle
```

Comment:
I had some weird behavior when tried `when-visible` e.g.
1. when-visible on sidebar
2. Resize to mobile and try click Filter btn
3. Page is unreactive - like crashed
4. Had to resize page
5. Sidebar appears

I was thinking about using the device module in default theme to get rid of some components on mobile. It might be a better approach

Reply:
maybe because sidebar require different action to be hydrated then intersectionObserver. In case of sidebar probably you should use `<LazyHydrate :trigger-hydration="isActiveSidebar">`. But those elements on PDP exist on initial load (html is generated on ssr) and we just need to scroll down to make them active.
device module also sounds ok. I will resolve it then

Reply:
Looks like this bug is not related to lazy hydration. So I applied more when-visible for Product.vue & trigger-hydration for mobile sidebar on category view